src/main/resources/lib/ssb/utils/calculatorUtils.ts

```diff
@@ -19,75 +16,63 @@ export function nextPeriod(month: string, year: string): CalculatorPeriod {
   }
 }
 
-export function allMonths(phrases: Phrases, frontPage?: boolean, type?: string): MonthDropdownItems {
-  const months: MonthDropdownItems = [
-    {
-      id: '01',
-      title: phrases.january,
-    },
-    {
-      id: '02',
-      title: phrases.february,
-    },
-    {
-      id: '03',
-      title: phrases.march,
-    },
-    {
-      id: '04',
-      title: phrases.april,
-    },
-    {
-      id: '05',
-      title: phrases.may,
-    },
-    {
-      id: '06',
-      title: phrases.june,
-    },
-    {
-      id: '07',
-      title: phrases.july,
-    },
-    {
-      id: '08',
-      title: phrases.august,
-    },
-    {
-      id: '09',
-      title: phrases.september,
-    },
-    {
-      id: '10',
-      title: phrases.october,
-    },
-    {
-      id: '11',
-      title: phrases.november,
-    },
-    {
-      id: '12',
-      title: phrases.december,
-    },
-  ]
-
-  if (type !== 'husleie' && type !== 'bkibol') {
-    const placeholderItem: MonthDropdownItem = {
-      id: '90',
-      title: frontPage ? phrases.calculatorMonthAverageFrontpage : phrases.calculatorMonthAverage,
-    }
+export function getMonthByQuartal(quarter: number | string) {
```

Comment:
Er denne funksjonen nødvendig, eller er det fordi man bruker samme valdidering for kvartal som får måned?

Reply:
Denne brukes kun av validering siden vi sjekker mot måned